pkgs/top-level/python-packages.nix

```
@@ -2937,6 +2937,8 @@ in {
 
   easygui = callPackage ../development/python-modules/easygui { };
 
+  EasyOCR = callPackage ../development/python-modules/EasyOCR { };
```

```suggestion
  easyocr = callPackage ../development/python-modules/easyocr { };
```
python attrs must be normalized